cfme/tests/ssui/test_ssui_service_catalogs.py

```diff
@@ -144,19 +146,51 @@ def test_ssui_disable_dashboard(appliance, user_self_service_role):
             assert not view.is_displayed
 
 
-@pytest.mark.manual
+@pytest.mark.customer_scenario
 @pytest.mark.tier(2)
-def test_sui_monitor_ansible_playbook_std_output():
-    """
-
+@pytest.mark.meta(automates=[1437210])
+def test_ssui_ansible_playbook_stdout(
+    appliance, ansible_catalog_item, ansible_service_request, ansible_catalog, request
+):
+    """ Test standard output of ansible playbook service
+    Bugzilla:
+        1437210
     Polarion:
         assignee: nansari
-        casecomponent: SelfServiceUI
-        testtype: functional
         initialEstimate: 1/4h
-        startsin: 5.8
-        tags: ssui
-    Bugzilla:
-        1437210
+        casecomponent: SelfServiceUI
+        setup:
+            1. Create ansible playbook
+        testSteps:
+            1. Create ansible playbook service catalog item
+            2. Create service catalog
+            3. Order service
+            4. Log in into SSUI
+            5. Navigate to My Services->Service details
+            6. check stdout of service
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5.
+            6. able to see standard output
     """
-    pass
+    service_catalog = ServiceCatalogs(appliance, ansible_catalog, ansible_catalog_item.name)
+    ansible_service = MyService(appliance, ansible_catalog_item.name)
+
+    @request.addfinalizer
+    def _finalize():
+        if ansible_service_request.exists():
+            ansible_service_request.remove_request()
+        if ansible_service.exists:
+            ansible_service.delete()
```

Comment:
Thoughts on using `ansible_service_catalog` and `ansible_service` fixtures instead of defining them here?